pkgs/by-name/fa/fastcov/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  python3Packages,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  libgcc,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "fastcov";
+  version = "1.16";
+
+  src = fetchFromGitHub {
+    owner = "RPGillespie6";
+    repo = "fastcov";
+    tag = "v${version}";
+    hash = "sha256-frpX0b8jqKfsxQrts5XkOkjgKlmi7p1r/+Mu7Dl4mm8=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    python3Packages.coverage
+    libgcc # provide gcov
+  ];
+
+  dontUseCmakeConfigure = true; # cmake is used for testing
+
+  nativeCheckInputs = with python3Packages; [
+    pytest
```

Comment:
`pytestCheckHook` seems to work for me, shouldn't we be using it instead of passing `pytest` directly?